src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -14,11 +14,42 @@ internal class ExternalXrefMapLoader
     {
         private static readonly byte[] s_uidBytes = Encoding.UTF8.GetBytes("uid");
 
-        public static IReadOnlyDictionary<string, Lazy<ExternalXrefSpec>> Load(Config config, FileResolver fileResolver)
+        public static IReadOnlyDictionary<string, Lazy<ExternalXrefSpec>> Load(
+            Config config,
+            FileResolver fileResolver,
+            Docset fallbackDocset,
+            Input input)
         {
             var result = new Dictionary<string, Lazy<ExternalXrefSpec>>();
 
             foreach (var url in config.Xref)
+            {
+                try
+                {
+                    LoadXref(url);
+                }
+                catch (DocfxException ex) when (ex.Error.Code == "file-not-found")
```

Comment:
Consider handle this at `FileResolver` layer to enable loc fallback for all contents referenced by config.

Reply:
Thought about that as well, did this the same behavior as `LinkResolver`
Now in `FileResolver`, it only holds `_docsetPath`, which can be default or fallback. And it is not singleton in our app, maybe we can make it singleton and hold fallback docset as well.

`DocumentProvider` is singleton in our app and holds default and fallback docset.
Maybe we can resolve this in `LinkResolver` as well.
https://github.com/dotnet/docfx/blob/8c969bfa998d5566e76ebf1c565fe7b0bcb148a8/src/docfx/build/dependency/LinkResolver.cs#L274

Prefer not to refactor them in this PR though

Reply:
`FileResolver` is used for resolving external link in config, move the fallback logic within it and would come up with a better naming for it, probably make it single instance as well later